app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/get_software.scala.html

```diff
@@ -0,0 +1,48 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.incometaxsubscriptioneligibilityfrontend.config.AppConfig
+@import uk.gov.hmrc.play.views.html.helpers.form
+@import uk.gov.hmrc.incometaxsubscriptioneligibilityfrontend.views.html._
+
+@(action: Call)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@main_template(
+title = Messages("get_software.title"),
```

Comment:
```suggestion
title = messages("get_software.title"),
```
Can this be changed to lowercase m so that it is consistent across the view?